src/Uno.UI.Tests/ResourceLoader/Strings/en/Resources.Designer.cs

```diff
@@ -61,29 +61,29 @@ internal class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to App70-en.
+        ///   Looks up a localized string similar to .
         /// </summary>
-        internal static string ApplicationName {
+        internal static string TestEmptyConverterText_ValueIfNotNull {
             get {
-                return ResourceManager.GetString("ApplicationName", resourceCulture);
+                return ResourceManager.GetString("TestEmptyConverterText.ValueIfNotNull", resourceCulture);
             }
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Text in &apos;en&apos;.
+        ///   Looks up a localized string similar to Test.
         /// </summary>
-        internal static string Given_ResourceLoader_When_LocalizedResource {
+        internal static string TestEmptyConverterText_ValueIfNull {
             get {
-                return ResourceManager.GetString("Given_ResourceLoader.When_LocalizedResource", resourceCulture);
+                return ResourceManager.GetString("TestEmptyConverterText.ValueIfNull", resourceCulture);
             }
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Header in &apos;en&apos;.
+        ///   Looks up a localized string similar to .
```

Comment:
similar to ?

Reply:
Those are generated files and the ordering has changed. We can ignore this.